src/explore-education-statistics-admin/src/services/publicationService.ts

```diff
@@ -135,9 +131,20 @@ const publicationService = {
     return client.get<MyPublication>(`/me/publication/${publicationId}`);
   },
 
-  getReleases(publicationId: string): Promise<ReleaseSummary[]> {
-    return client.get<ReleaseSummary[]>(
+  getReleases(
+    publicationId: string,
+    page: number | undefined = undefined,
+    pageSize: number | undefined = undefined,
+    live: boolean | undefined = undefined,
+  ): Promise<PaginatedList<ReleaseListItem>> {
+    let params = {};
+    if (page !== undefined) params = { ...params, page };
+    if (pageSize !== undefined) params = { ...params, pageSize };
+    if (live !== undefined) params = { ...params, live };
+
+    return client.get<PaginatedList<ReleaseListItem>>(
       `/publication/${publicationId}/releases`,
+      { params },
```

Comment:
This could be simplified by making the params an optional object that gets passed in, something like:
```
export interface getReleasesParams {
  live?: boolean;
  page?: number;
  pageSize?: number;
}

 getReleases(
    publicationId: string,
    params?: getReleasesParams,
  ): Promise<PaginatedList<ReleaseListItem>> {
    return client.get<PaginatedList<ReleaseListItem>>(
      `/publication/${publicationId}/releases`,
      { params },
    );
  },
  ```
  
  which could then be called with or without the params:
  ```
  await publicationService.getReleases(
      publicationId,
      { page: 2 },
    );
   
    await publicationService.getReleases(publicationId);